compiler/test/dotty/tools/dotc/printing/SyntaxHighlightingTests.scala

```diff
@@ -23,7 +23,9 @@ class SyntaxHighlightingTests extends DottyTest {
 
     if (expected != highlighted) {
       // assertEquals produces weird expected/found message
-      fail(s"expected: $expected but was: $highlighted")
+      fail(s"""|
```

Comment:
Not related to the actual changes, but this makes seeing the actual diff much easier when testing.